src/main/java/com/enfint/dealservice/service/impl/CalculationServiceImpl.java

```diff
@@ -0,0 +1,108 @@
+package com.enfint.dealservice.service.impl;
+
+import com.enfint.dealservice.dto.CreditDTO;
+import com.enfint.dealservice.dto.LoanOfferDTO;
+import com.enfint.dealservice.dto.ScoringDataDTO;
+import com.enfint.dealservice.entity.Application;
+import com.enfint.dealservice.entity.Client;
+import com.enfint.dealservice.entity.Credit;
+import com.enfint.dealservice.exception.ApplicationNotFoundException;
+import com.enfint.dealservice.feignclient.ConveyorClient;
+import com.enfint.dealservice.repository.ApplicationRepository;
+import com.enfint.dealservice.repository.ClientRepository;
+import com.enfint.dealservice.repository.CreditRepository;
+import com.enfint.dealservice.service.CalculationService;
+import com.enfint.dealservice.utils.ApplicationStatusEnum;
+import com.enfint.dealservice.utils.CreditStatusEnum;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.modelmapper.ModelMapper;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+@Slf4j
+public class CalculationServiceImpl implements CalculationService {
+
+    private final ApplicationRepository applicationRepository;
+    private final CreditRepository creditRepository;
+    private final ClientRepository clientRepository;
+    private final ConveyorClient conveyorClient;
+    private final ModelMapper mapper;
+
+    @Override
+    public void getApplicationById(ScoringDataDTO scoringData, Long applicationId) {
```

Comment:
method which called "*get*Something" should return something. Rename it or return application